src/app/app.module.ts

```diff
@@ -1,7 +1,7 @@
 // core
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-import { FormsModule } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
```

Comment:
Let's look at this when you move Seeyond in to see if it's better than our template validation we have now.